portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,26 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+//Fetch request from the page /comments and put returned info to the list
+function showComments() {
+    console.log("fetching comments");
+    fetch("/comments").then(response => console.log(response.json().name.text()));
+
+    fetch("/comments").then(response => response.json()).then((stats) => {
+      const statsListElement = document.getElementById('comment-field');
+      statsListElement.innerHTML = '';
+      for (let i = 0; i < 3; i++) {
```

Comment:
This should use the array length rather than the hardcoded value of 3 as an upper bound.